tests/sentry/search/events/test_filter.py

```diff
@@ -1206,11 +1225,7 @@ def test_function_with_negative_arguments(self):
     def test_function_with_bad_arguments(self):
         result = get_filter("percentile(transaction.duration 0.75):>100")
         assert result.having == []
-        assert result.conditions == [
-            _om("percentile"),
-            _om("transaction.duration 0.75"),
-            _om(":>100"),
-        ]
+        assert result.conditions == [_om("percentile(transaction.duration 0.75):>100")]
```

Comment:
I'm excited for a syntax highlighter that will hopefully help users understand why this query doesn't return what they expect.